src/main/java/com/gluonhq/substrate/Constants.java

```diff
@@ -102,7 +102,7 @@
     public static final String PROFILE_LINUX_AARCH64 = "linux-aarch64";
 
     // public static final String DEFAULT_JAVA_STATIC_SDK_VERSION  = "15-ea+4";
-    public static final String DEFAULT_JAVA_STATIC_SDK_VERSION  = "11-ea+1";
+    public static final String DEFAULT_JAVA_STATIC_SDK_VERSION  = "11-ea+10";
```

Comment:
Do we only use this on ios/android (and not on linux-aarch64 for example?)

Reply:
You are right, we need the linux-aarch64 build too:
```
Downloading failed: https://download2.gluonhq.com/substrate/staticjdk/labs-staticjdk-linux-aarch64-gvm-11-ea+10.zip
```

Reply:
Ok, link works now for linux-aarch64